src/main/java/org/traccar/database/NotificationManager.java

```diff
@@ -53,6 +53,8 @@ public class NotificationManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NotificationManager.class);
 
+    private static final String ATTRIBUTE_DEVICE_ALLOW_FORWARDING = "deviceAllowEventForwarding";
```

Comment:
Let's rename to "enableEventForwarding".

Reply:
Are you sure as all other attributes start with deviceXxxx?

Reply:
Yes, I am sure.